runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -782,9 +787,18 @@ public void setResourceClient(XWalkResourceClientInternal client) {
      */
     @Override
     @XWalkAPI
-    public void setBackgroundColor(int color) {
+    public void setBackgroundColor(final int color) {
```

Comment:
Okay, i see `src/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromium.java` did the same thing.